api/lib/accounts.js

```diff
@@ -34,6 +34,25 @@ const registerUser = async (uid) => {
   }
 }
 
+const sessionCache = {}
+async function validateSession(sessionToken) {
+  let session
+  if (sessionCache[sessionToken]) {
+    session = sessionCache[sessionToken]
+  } else {
+    session = await db.getSession(sessionToken)
```

Comment:
the Lunie sessions are long-lived right?

```
 SESSION {
  uid: 'R5KcpKptl0MaFWeiGBcDNzY2BSL2',
  valid_until: '2020-09-01T10:32:51.49+00:00'
}
```

Reply:
30 days. We can shorten that.